internal/oidc/verifier_test.go

```
@@ -369,3 +379,410 @@ func makeJWTHeader(header map[string]any) string {
+	}
+	if _, err := mgr.createHTTPClientFromTokenPath(config.ClusterConfig{CACert: badCA}); err == nil {
+		t.Error("expected error for unparsable CA cert")
+	}
+}
+
+func TestTokenRoundTripperMissingFile(t *testing.T) {
+	client := &http.Client{Transport: &tokenRoundTripper{
+		transport: http.DefaultTransport,
+		tokenPath: t.TempDir() + "/absent",
+	}}
+	if _, err := client.Get("http://127.0.0.1:1/"); err == nil {
```

golangci-lint (2.12.2)

[error] response body must be closed

(bodyclose)